plugins/runtime/example/main.go

```diff
@@ -32,11 +32,11 @@ func initMeter() *push.Controller {
 	pusher, err := metricstdout.NewExportPipeline(metricstdout.Config{
 		Quantiles:   []float64{0.5},
 		PrettyPrint: true,
-	}, 10*time.Second)
+	})
```

Comment:
```suggestion
	}, push.WithPeriod(10*time.Second)
```

Should this still specify a period?

Reply:
I changed the default here because it was too long to wait for an example.  It made me think the example wasn't working because stdout was blank.

Reply:
The default has changed to 10s, so I removed the explicit parameter to keep an unnecessary detail out of the example.